test/parallel/test-tls-js-stream.js

```diff
@@ -15,8 +16,8 @@ const connected = {
 };
 
 const server = tls.createServer({
-  key: fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`),
-  cert: fs.readFileSync(`${common.fixturesDir}/keys/agent1-cert.pem`)
+  key: fixtures.readFixtureKey('agent1-key.pem'),
+  cert: fixtures.readFixtureKey('agent1-cert.pem')
```

Comment:
these can be:
```
key: fixtures.readSync('agent1-key.pem'),
cert: fixtures.readSync('agent1-cert.pem')
```

Reply:
@gireeshpunathil  (and for @wsierakowski's benefit too): I think you mean `readKey()` rather than `readSync()`. (Aside: API should have probably been `readKeySync()` rather than `readKey()` but whatever. :-D )

Reply:
@Trott - yes, thanks for the correction!